monitoring/dashboards/caretta.yaml

```diff
@@ -0,0 +1,499 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: monitoring
+  name: "caretta-dashboards"
+  annotations:
+     grafana_folder: "Caretta service graph dashboards"
+  labels:
+    grafana_dashboard: "1"
+    app: observability-grafana
+data:
+ applications.json: |- 
+    {
```

Comment:
Please demo this dashboard before I can merge, also as I mentioned this should be part of the main application dashboard.